views/js/runner/plugins/content/accessibility/keyNavigation.js

```diff
@@ -137,40 +147,41 @@ define([
             //update plugin state based on changes
             testRunner
                 .on('renderitem', function () {
+
                     var $content = testRunner.getAreaBroker().getContentArea();
+                    var index = 1;
 
-                    responses = $(':input', $content).toArray();
-                    responses.sort(function (a, b) {
-                        var aIndex = a.tabIndex || 1;
-                        var bIndex = b.tabIndex || 1;
-                        return parseInt(aIndex, 10) - parseInt(bIndex, 10);
-                    });
-                    count = responses.length || 1;
+                    focusables = $(':input,img', $content).addClass('key-navigation-focusable').each(function(){
+                        var $this = $(this);
+                        //redistribute focus order
+                        $this.attr('tabindex', index);
+                        index++;
+                    }).toArray();
+
+                    count = focusables.length || 1;
                     cursor = null;
                     self.enable();
                 })
                 .on('unloaditem', function () {
                     self.disable();
                 })
-                .on('previous-response', function() {
+                .on('previous-focusable', function() {
+                    console.log('aaa');
                     if (self.getState('enabled')) {
-                        previousResponse();
+                        previousFocusable();
                     }
                 })
-                .on('next-response', function() {
+                .on('next-focusable', function() {
                     if (self.getState('enabled')) {
-                        nextResponse();
+                        nextFocusable();
                     }
                 });
         },
 
         /**
          * Called during the runner's render phase
          */
-        render: function render() {
-            var $container = this.getAreaBroker().getToolboxArea();
-            $container.append(this.$button);
-        },
+        render: _.noop,
```

Comment:
Not sure this is needed. You can just remove the declaration of this method. If it is missing this won't trigger any error as the call will simply be ignored (delegation).